tests/appveyor/mssql.appveyor.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="../../vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+
+        <var name="db_type" value="pdo_sqlvr"/>
```

Comment:
You can look up the driver names in the Driver classes like [here]().
A complete list of supported drivers currently is:

- `ibm_db2`
- `mysqli`
- `oci8`
- `pdo_mysql`
- `pdo_oracle`
- `pdo_pgsql`
- `pdo_sqlite`
- `pdo_sqlsrv`
- `sqlanywhere`
- `sqlsrv`

Simply leave those out that cannot (yet) be run on AppVeyor.

If different database versions and or vendors (MySQL/MariaDB) require different connection parameters, we should probably extend the convention to something like `(<db_name>(.<db_version>)?.)?<dbal_driver_name>.appveyor.xml`